RELEASE.md

```diff
@@ -12,19 +12,12 @@
 * Upgraded `pip-tools`, which is used by `kedro build-reqs`, to 6.4. This `pip-tools` version requires `pip>=21.2` while [adding support for `pip>=21.3`](https://github.com/jazzband/pip-tools/pull/1501). To upgrade `pip`, please refer to [their documentation](https://pip.pypa.io/en/stable/installing/#upgrading-pip).
 * Extended ``ExcelDataSet`` to support saving Excel files with multiple sheets.
 * `kedro pipeline package <pipeline>` now raises an error if the `<pipeline>` argument doesn't look like a valid Python module path (e.g. has `/` instead of `.`).
+* Removed "Thanks for supporting contributions" section on Release notes to simplify the contribution process
```

Comment:
Prototype bash command:

```bash
g log "$(git tag -l --sort=-creatordate | head -n 1)".."HEAD" --no-merges "$@" \ 
    | grep ^Author: \ 
    | sed 's/ <.*//; s/^Author: //' \ 
    | sort -u \ 
    | sed -e 'H;${x;s/\n/, /g;s/^,//;p;};d' \ 
    | awk '{$1=$1};1' \ 
    | sed -e 's/^/A special thank you to our contributors since the last release (auto generated from the `git log`): \n/' 

```

1. Get git log between last release and current `HEAD`
2. Find rows where Author mentioned
3. Extract name from Author label
4. Get unique names
5. Collect into comma separated string
6. Trim whitespace
7. Prepend with thank you message

Results look like follows:

```
A very special thank you to our contributors since the last release (auto generated from the `git log`): 
aaaa, bbb, ccc
```